docs/document/docs/guides/tabs-customzation.md

```diff
@@ -2,34 +2,35 @@
 id: tabs-customization
```

Comment:
There is no JA version doc. Please add it.

Reply:
i have fixed